.prettierignore

```diff
@@ -1,4 +1,5 @@
 public/build/bundle.js
 public/build/bundle.css
 public/data/*
+public/build/public/build/bundle.css
```

Comment:
That path shouldn't exist. The `bundle.css` should only be written to `public/build/bundle.css`. I believe this was an error in the earlier app template that was [fixed](5e787003843d662079e8c047078c108de8b6d976).

Reply:
Thanks, I think we need to update the rollup.config.js so that the tailwind minified bundle use a different name. There is already one with the same name `public/build/bundle.css`. 

By the way, the link to the fix isn't working.

Reply:
Updated the link. I guess the github linker to a commit hash thought it was a pull request. Weird.